src/hyperlane-adapter/HyperlaneReceiverAdapter.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.16;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { IMailbox } from "./interfaces/IMailbox.sol";
+import { IMessageRecipient } from "./interfaces/IMessageRecipient.sol";
+import { IInterchainSecurityModule, ISpecifiesInterchainSecurityModule } from "./interfaces/IInterchainSecurityModule.sol";
+import { TypeCasts } from "./libraries/TypeCasts.sol";
+import { Errors } from "./libraries/Errors.sol";
+import { IMessageDispatcher } from "./interfaces/IMessageDispatcher.sol";
+import { IMessageExecutor } from "./interfaces/IMessageExecutor.sol";
+import "./libraries/MessageLib.sol";
+
+/**
+ * @title HyperlaneReceiverAdapter implementation.
+ * @notice `IBridgeReceiverAdapter` implementation that uses Hyperlane as the bridge.
+ */
+contract HyperlaneReceiverAdapter is
```

Comment:
Can you rename this to indicate that this only supports v2

Reply:
Fixed